Casks/abbyy-finereader-pdf.rb

```diff
@@ -0,0 +1,43 @@
+cask "abbyy-finereader-pdf" do
+  version "1402.9"
+  sha256 :no_check
+
+  url "https://downloads.abbyy.com/fr/fr_mac/current/ABBYY_FineReader_PDF.dmg?secure=5c11NlPrQmIZeGn6dK2PZA=="
+  name "ABBYY FineReader PDF"
+  desc "Scan, OCR, and convert documents to searchable PDFs and other formats"
+  homepage "https://pdf.abbyy.com/finereader-pdf-for-mac/"
+
+  livecheck do
+    url "https://www.abbyy.com/checkforupdates/?Product=FineReaderMac&Target=CheckUpdate&PartNumber=#{version}&Language=en"
+    strategy :page_match do |page|
+      page.scan(%r{(?:Update Part Number: |part# <b>)(\d+(?:[./]\d+)+)}i)
+          .map { |match| match[0].tr("/", ".") }
+    end
+  end
+
+  app "ABBYY FineReader PDF.app"
+
+  zap trash: [
+    "~/Library/Application Support/ABBYY",
+    "~/Library/Preferences/com.abbyy.finereader.pdf.plist",
+    "~/Library/Preferences/com.abbyy.FineReader.plist",
+    "~/Library/Preferences/com.abbyy.FRPDFViewer.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService1.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService2.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService3.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService4.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService5.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService6.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService7.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService8.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService9.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService10.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService11.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService12.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService13.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService14.plist",
+    "~/Library/Preferences/com.abbyy.ImageProcessService15.plist",
```

Comment:
```suggestion
    "~/Library/Preferences/com.abbyy.ImageProcessService*.plist",
```
Roll them all into one with the `*` wildcard.

Reply:
Great! I was wondering whether the zap stanza supported wildcards, but it wasn't covered in the documentation (at least for the zap stanza specifically).

Reply:
Ok, I've applied the wildcard.